### PR TITLE
# updated the link as the previous one was corrupted

### PR DESCRIPTION
**Pull request by csmailis**
# updated the selective_search_data link as the previous one was corrupted
the new link is wget https://dl.dropboxusercontent.com/s/orrt7o6bp6ae0tc/selective_search_data.tgz

## Description ##
(Brief description on what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by marcoabreu** [comment]
Thank you for providing the non-corrupt file Christos. This is a great effort to increase the stability of our examples. To ensure we're keeping the files for a long time, we'd like to re-upload it to an S3 bucket which is under our control.

Also, do you know the license for this data set? Are we allowed to redistribute it?

File: example/rcnn/script/get_selective_search.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ fi
 pushd data
 
 # the result is selective_search_data
-wget http://www.cs.berkeley.edu/~rbg/fast-rcnn-data/selective_search_data.tgz
-tar xf selective_search_data.tgz
+wget https://dl.dropboxusercontent.com/s/orrt7o6bp6ae0tc/selective_search_data.tgz
```
Comment (szha) [inline example/rcnn/script/get_selective_search.sh]: you can use https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/gluon/dataset/selective_search_data.tgz

**Comment by eric-haibin-lin** [comment]
could you resolve conflict?

**Comment by lupesko** [comment]
@csmailis bouncing this: can you please update the location and resolve conflicts? thanks!

**Comment by lupesko** [comment]
@csmailis bounce...

**Comment by stu1130** [comment]
bounce again @csmailis please address the conflict and update URL. Thanks

**Comment by vandanavk** [comment]
This PR is still awaiting response - continuing with the label.

**Comment by Roshrini** [comment]
@csmailis @sandeep-krishnamurthy Can you please resolve conflict so that we can move forward with this PR? Thanks!

**Comment by ankkhedia** [comment]
@csmailis @sandeep-krishnamurthy Could you please resolve conflicts so that the PR can be merged.

**Comment by sandeep-krishnamurthy** [comment]
@ankkhedia - I suggest to take over the changes in this PR, as it looks like stale, but with imp change, and submit a fix.

**Comment by ankkhedia** [comment]
@sandeep-krishnamurthy Sure thing! I will take a look.

**Comment by sandeep-krishnamurthy** [comment]
Closing the PR as this file does not exist anymore and the corrected link is no more referenced in the repo.

**Pull request closed without merging**